### ISSUE TITLE
Live TV streams (TV 1, TV 2, Teema) do not play outside Finland

### ISSUE THREAD

**Issue by Mateus109** [aajanki/plugin.video.yleareena.jade#8]
Playback seems to have stopped working. None of the three Yle channels will play. Have Yle blocked the API?

**Comment by aajanki** [comment]
Are there any errors on the log?

If you are located outside of Finland, the channels won't play because they are geo-blocked.

**Comment by Mateus109** [comment]
Yes, I'm located outside of Finland, but this plugin used to bypass the geo-block. We been using it for a few years. Or maybe I'm getting confused with a different Kodi plug-in for Yle!
